pkgs/development/python-modules/humanize/default.nix

```
@@ -4,15 +4,17 @@
   fetchFromGitHub,
   freezegun,
   gettext,
+  pytest-cov,
   pytestCheckHook,
   pythonOlder,
+  python3,
```

```suggestion
  python,
```
otherwise we mix pythons